src/grid.c

```diff
@@ -1135,8 +1135,7 @@ SEXP L_convert(SEXP x, SEXP whatfrom,
          * In these cases do NOT transform thru INCHES 
          * (to avoid divide-by-zero, but still do something useful)
          */
-        relConvert = (!isUnitArithmetic(x) && !isUnitList(x) &&
-                      (unitUnit(x, i) == L_NATIVE || unitUnit(x, i) == L_NPC) &&
+        relConvert = ((unitUnit(x, i) == L_NATIVE || unitUnit(x, i) == L_NPC) &&
```

Comment:
Does this change (dropping the unitArithmetic and unitList) depend on your other changes to unit code ?

Reply:
Huh, weird - that shouldn't be in this pr... thanks for catching that!